### ISSUE TITLE
Final Task List 

### ISSUE THREAD

**Issue by Bshortall** [BostonUniversitySeniorDesign/brain-4ce#20]
**Team necessities (ALL)**
- [x] Ask Osama for GPU + Monitors for ECE day
- [x] Print poster (4/28)
- [x] Resource GitHub (4/30)
- [ ] Test end-to-end system
- [ ] Make decision on if we need two instances of brainflow code, one for each game. I am thinking B.F. gets input from V.E. that tells it if it should send data every 3s or .5s, model gets same input to decide which algorithm to use. 

**Models  (Brendan)**
- [ ] Finish concentration algorithm
- [ ] Determine what rows are needed from raw B.F. data 
- [ ] Finetune using our data
- [ ]  Tune hyperparameters (if time permitting)

**Sockets (Dayanna)**
- [ ]  Connect B.F. -> Model -> Virtual Environment
- [ ] Model output has two values, prediction and concentration level
- [ ]  Connect Virtual Environment -> B.F. (Tell when to start and stop streaming) and model ?

**Virtual Environment (Jonathan)**
- [x] Home screen to start either game
- [ ] Scoreboard
- [x] Instructions page (for each game)
- 3D  Game
- [ ] Decide how to smooth movement (gets input from model every ~3s)
- 2D Game
- [ ] decide how input from brainflow will correlate to movement of the bar 
- 30s game
- [x] Every 10s move green bar to random position
- [x]  x points for every second in green bar

**Data Collection (Mitch & Jon)**
- [x] Record for entirety of GUI display
- [x] Use timestamps of GUI to label 
   - Each sample should be 1s of baseline (no action showing) followed by 2 seconds of action.
- [ ] Modify total data into 3D array 

**Hardware (Alex & Mitch)**
- [ ] USB Isolator ?
- [ ] Ear Clips
- [ ] New headset, includes  placing probes and possibly new case.
